### DISCUSSION TITLE
Can I manually write a Header or Record?

### DISCUSSION THREAD

**Discussion by niko86** [JoshClose/CsvHelper#1833] [Q&A] [answered]
Hello,

The title covers it all really. When reading I can use the Parser to get RawRecord or Record etc. without mapping to a class object.

Is there something analogous for CsvWriter?

Thanks

**Comment by JoshClose** [comment, marked as answer]
You can use `WriteField` to write individual fields. You need to call `NextRecord` to complete the row when writing manually.

**Reply by niko86** [reply]
Thank you Josh for the quick reply and apologies that I missed the WriteField method.
